src/startupmgr/.manifest

```diff
@@ -0,0 +1,3 @@
+{
+	description = "Execute all scripts in the /startup.d folder"
```

Comment:
I'm wondering whether we should merge `startupmgr` with `autostartup`. Thoughts?

Reply:
Maybe. What are their differences?